daemon/api_notices.go

```diff
@@ -27,11 +27,16 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
+var noticeCheckers = map[state.NoticeType]accessChecker{
+	state.ChangeUpdateNotice: interfaceOpenAccess{Interface: "snap-refresh-observe"},
+	state.WarningNotice:      openAccess{},
```

Comment:
we strictly don't need this entry as it's the default?

Reply:
@olivercalder yes exactly